python/src/spark_rapids_ml/classification.py

```diff
@@ -976,6 +976,18 @@ def _single_fit(init_parameters: Dict[str, Any]) -> Dict[str, Any]:
                     "dtype": logistic_regression.dtype.name,
                     "num_iters": logistic_regression.solver_model.num_iters,
                 }
+
+                if len(logistic_regression.classes_) == 1:
+                    class_val = logistic_regression.classes_[0]
+                    assert (
+                        class_val == 1.0 or class_val == 0.0
+                    ), "class value must be either 1. or 0. when dataset has one label"
```

Comment:
What does spark do if label has one value but is not 1 or 0?

Reply:
Revised. 
if label < 0, a java runtimeError pops up. 
If label > 1, spark trains a multinomial classification, cuml trains a single-class classification due to using y.unique().